_posts/2020-04-17-quarkus-mutual-tls.adoc

```diff
@@ -0,0 +1,405 @@
+---
+layout: post
+title: 'Quarkus - Makes Mutual TLS Easy.'
+date:   2020-04-17 00:00 +0100
+author: mkouba
+synopsis: Quarkus - Mutual TLS Authentication.
+---
+
```

Comment:
In this entry, we will explore how to manually set mutual TLS encryption between two Quarkus applications.
Both on bare metal and on Kubernetes.